ccmbioinfo/lfs
# LFS-1157: License information appears indexed as a term in ORDO

This PR addresses the LFS-1157 issue by checking the `/VocabulariesIgnore/<VOCAB-NAME>` sub-tree to obtain the list of vocabulary term nodes that are to be ignored when installing `<VOCAB-NAME>` where `<VOCAB-NAME>` is a OWL formatted vocabulary installed from BioPortal.

To test:

- Build and start the `dev` branch.
- Set a valid BioPortal API key and install the ORDO vocabulary from BioPortal
- After vocabulary installation, visit `http://localhost:8080/Vocabularies/ORDO/4.0.info.json`. This should display the JSON serialization for the _CC BY 4.0_ Creative Commons License term from the vocabulary.

---

- Stop CARDS and `rm -rf sling`

---

- Now build and start the `LFS-1157` branch.
- Set a valid BioPortal API key and install the ORDO vocabulary from BioPortal
- After vocabulary installation, visit `http://localhost:8080/Vocabularies/ORDO/4.0.info.json`. This should display a 404 error page.
- Install the HANCESTRO vocabulary from BioPortal
- Experiment with Forms that use the _ORDO_ and _HANCESTRO_ vocabularies. All should work as expected.
